### ISSUE TITLE
can not compile

### ISSUE THREAD

**Issue by ghost** [appknox/androidvncserver#2]
```
vnc/droidvncserver.c:21:10: fatal error: 'common.h' file not found
#include "common.h"
         ^~~~~~~~~~
2 warnings and 1 error generated.
make: *** [obj/local/armeabi-v7a/objs/androidvncserver/droidvncserver.o] Error 1
```

**Comment by Praseetha-KR** [comment]
@di3663 `include` folders were missing, which is fixed in 5f150c00c5ddda08c3d2ee5515f038860e776f21. Can you pull the latest and check?

**Comment by ghost** [comment]
works now.
